projects/demo-integrations/cypress/tests/desktop/core/marker-icon/marker-icon.cy.ts

```diff
@@ -11,18 +11,25 @@ describe(`tuiMarkerIcon`, () => {
         cy.get(`@content`).find(`a[tuiMarkerIcon]`).as(`markerIcon`);
 
         cy.get(`@content`).matchImageSnapshot(`tui-all-icons`);
-        /**
-         * @description:
-         * https://github.com/dmtrKovalenko/cypress-real-events#1-why-cyrealhover-hovering-state-does-not-show-in-the-visual-regression-services
-         * Unfortunately, neither visual regression services like Happo and Percy nor plain cy.screenshot
-         * do not allow to test the hovering state. The hovering state is very different from any kind of js
-         * and css so it is not possible to capture it using dom snapshotting
-         * (like visual regression services do) and the screenshotting as well because
-         * cypress core itself is preventing hovering state in the screenshots.
-         */
-        cy.get(`@markerIcon`)
-            .should(`have.css`, `background-color`, `rgb(235, 239, 255)`)
-            .realHover()
-            .should(`have.css`, `background-color`, `rgb(223, 227, 243)`);
+
+        cy.get(`body`)
+            .invoke(`attr`, `data-tui-theme`)
+            .then(theme => {
+                if (theme === `taiga`) {
```

Comment:
`.should(`have.css`, `background-color`, `rgb(235, 239, 255)`)` not valid for tinkoff theme